src/components/views/Dashboard404Report/Dashboard404TableView.jsx

```diff
@@ -59,23 +64,12 @@ import {
                                         );
                                       })
                                   )}
-                              </Row>
-                            )
+                              </Row>)
                       }
                   </TableBody>
-              </TableView>
-      );
-    } if (dataFlag) {
-      return (
-              <ProgressBar margin="auto" label="Loading…" isIndeterminate />
-      );
-    }
-    return (
-              <IllustratedMessage margin="auto">
-                  <NotFound />
-                  <Heading>No results</Heading>
-                  <Content>Use Filters</Content>
-              </IllustratedMessage>
+              </TableView>)
+        : <NoDataSkeleton />
```

Comment:
The idea is to only show "No Data" to end user if there's no data after fetching, otherwise display a fetching state.
Showing no data before the data is fetched can be misleading.